### PR TITLE
jquery validation into the skin branch

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: CRP.Mvc/App_Start/BundleConfig.cs [hunk 1]
```diff
@@ -14,8 +14,12 @@ public static void RegisterBundles(BundleCollection bundles)
             bundles.Add(new ScriptBundle("~/bundles/jqueryCurrent").Include(
                         "~/Scripts/jquery-1.10.2.js"));
 
-            bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
-                        "~/Scripts/jquery.validate*"));
+            //Needs <script src="https://ajax.microsoft.com/ajax/jquery.validate/1.8.1/jquery.validate.min.js"></script>
+            bundles.Add(new ScriptBundle("~/bundles/checkout").Include(
+                        "~/Scripts/xVal.jquery.validate.js",
```
Comment (srkirkland) [inline CRP.Mvc/App_Start/BundleConfig.cs]: @jSylvestre we're not really using xVal are we?  That has been deprecated for half a decade and their website says don't use it with anything after MVC 1.

File: CRP.Mvc/Views/Transaction/CheckoutNew.cshtml [hunk 2]
```diff
@@ -150,44 +150,48 @@
...
-                @{
-                    var questionSetName = qs.QuestionSet.Name;
-                    if (questionSetName == "Contact Information")
-                    {
-                        questionSetName = questionSetName + " / Billing Information";
-                        showDonationLink = true;
-                    }
-                    else
+            <div class="TransactionContainer">
+                <span id='@Html.Encode(qs.Id)'>
```
Comment (srkirkland) [inline CRP.Mvc/Views/Transaction/CheckoutNew.cshtml]: @jSylvestre You don't need to explicitly html.encode anymore, it's the default now

File: CRP.Mvc/Views/Transaction/CheckoutNew.cshtml [hunk 3]
```diff
@@ -150,44 +150,48 @@
...
+                    @{
+                        var questionSetName = qs.QuestionSet.Name;
+                        if (questionSetName == "Contact Information")
```
Comment (srkirkland) [inline CRP.Mvc/Views/Transaction/CheckoutNew.cshtml]: @jSylvestre if this works I'm fine with it but I do like the Ordinal string.Equals checks just to clarify what kind of string compare you are using.

File: CRP.Mvc/App_Start/BundleConfig.cs (hunk 1 above)
Comment (jSylvestre) [inline CRP.Mvc/App_Start/BundleConfig.cs]: @srkirkland Yes, yes we are.

**Pull request merged**